.github/dependabot.yml

```diff
@@ -21,9 +22,11 @@ updates:
       - ccamel
       - fredericvilcot
       - ErikssonJoakim
+      - lolottetheclash
     assignees:
       - ccamel
       - fredericvilcot
+      - lolottetheclash
     open-pull-requests-limit: 5
   - package-ecosystem: docker
```

Comment:
If you don't mind, could please force `dependabot` configuration to only open PR about `docker` related to patches? Thanks 😎

Reply:
@fredericvilcot I prefer to keep Dependabot PRs for all Docker images. This way, we can stay up-to-date with all available upgrades and security patches, especially regarding the `nginx` docker image we use.

Reply:
All right. Thanks.